src/server/index.js

```diff
@@ -4,18 +4,22 @@ import { MCFAdapter as MCFAdapterClass } from './MCFadapter'
 import { TCGAAdapter as TCGAAdapterClass } from './TCGAadapter'
 import bodyParser from 'body-parser'
 import Pool from 'pg-pool'
+import MongoClient from 'mongodb'
 
 const app = express()
 const port = process.env.PORT || 8080
 const dbURL = new URL(
   process.env.DATABASE_URL || 'postgres://genomedb:genomedb@postgres:5432/genomedb'
 )
+const mongodbURL = process.env.MONGODB_URL
+const mongodbDB = process.env.MONGO_DATABASE
 
 const MCFAdapter = new MCFAdapterClass()
 const TCGAAdapter = new TCGAAdapterClass()
 
 let adapter // set by middleware
 let pool
```

Comment:
Make sure to remove any postgres related initialization code since we don't need it.